database/seeders/GenreSeeder.php

```diff
@@ -13,10 +14,8 @@ class GenreSeeder extends Seeder
      */
     public function run(): void
```

Comment:
You actually might not need this seeder, or the review seeder any more - since the book factory creates its own genres and reviews